text/0000-fabric-private-chaincode-1.0.md

```diff
@@ -226,8 +226,9 @@ Hence all data received via transaction invocations (e.g., the transaction propo
 - We additionally require that remote attestations provided by a TEE are authentic and prove that only the code referenced in the attestation could have issued it.
 Therefore, all participants/organizations trust a TEE (in particular, the FPC Chaincode Enclave), which can provide such an attestation, regardless of at which peer/organization the TEE is hosted.
 Such trust extends also to enclave signatures issued from attested cryptographic keys.
-It is due to such trust that FPC *implicitly* defines an enclave endorsement policy,
+It is due to such trust that FPC *implicitly* defines an enclave endorsement policy (see also the [FPC Management API](https://github.com/hyperledger-labs/fabric-private-chaincode/blob/master/docs/design/fabric-v2%2B/fpc-management.md) document),
 according to which a single enclave signature (enclave endorsement) is sufficient to [validate](#enclave-endorsement-validation) an FPC transaction successfully.
+For clarity, the (FPC) enclave endorsement policy is different than the (Fabric) validation endorsement policy, the latter of which should be [determined](#enclave-endorsement-validation) from the organizational trust relationships in the Fabric network.
```

Comment:
We don't define `validation endorsement policy` and the "(fabric)' might confuse. Readers might think fabric defines such (which it doesn't, this is an FPC term) rather than what you probably meant that this FPC term is realized as a normal fabric endorsement policy.   More importantly, though. it seems to me this is a bit the wrong place to discuss this distinction as it lacks context. I think this would be better (as easier to understand given the context) discussed in the (first) "Enclave Endorsement Validation" section.  The general problem is of course that we discuss threat model at an architectural and not only conceptual level but do that before we define architecture. As we were asked to put threat model early. I think we probably should not discuss anything specific here (i.e., anything after "It is due to such trust ...") but replace it with a simple reference to the "Enclave Endorsement Validation" section on how this relates to endorsement polices?

I also think the specific reference to the section in fpc-management.md i had in my original would be better than just the document reference as you have)